next/components/main-menu/main-menu.components.tsx

```diff
@@ -177,6 +184,7 @@ export function MenuTrigger({ isTopLevel }: { isTopLevel?: boolean }) {
           ? "lg:ring-white lg:aria-expanded:bg-white lg:aria-expanded:text-primary-600"
           : "lg:aria-expanded:bg-primary-600 lg:aria-expanded:text-white lg:aria-expanded:ring-primary-600"
       )}
+      aria-label={`${t("show-submenu")}${" "}${t(parent)}`}
```

Comment:
This looks good, but we don't need to run `parent` through `t()` because it's already translated in Drupal side.

Proposed solution: in common.json, we can have `"show-submenu": "Show submenu for {{parent}}",`, and then we can use it like `aria-label={t("show-submenu", { parent })}` (see the translation `form-submission-intro-text` for another example)

Reply:
This is pure gold! Cheers for this!